source/includes/_core_agent.md

```diff
@@ -4,13 +4,98 @@ Some of the languages instrumented by Scout depend on a standalone binary for co
 
 If the Core Agent is required for your language, the Scout agent library for that language will handle downloading, configuring, and launching the Core Agent automatically. However, you may manually manage the Core Agent through configuration options.
 
+## Launching Core Agent manually
+
+<table class="help">
+  <tbody>
+    <tr>
+      <td>
+        <span class="step">1</span>
+      </td>
+      <td>
+        <p>Create a directory which your app has access to (for example: /var/www)</p>
+<pre style='width:500px'>
+cd /var/www
+mkdir scout_apm_core
+</pre>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <span class="step">2</span>
+      </td>
+      <td>
+        <p>Download and test the core agent:</p>
+<pre style='width:500px'>
+# 1. cd into the scout_apm_core directory
+cd ./scout_apm-core
+
+# 2. Download the core agent tarball
+curl https://s3-us-west-1.amazonaws.com/scout-public-downloads/apm_core_agent/release/scout_apm_core-latest-x86_64-unknown-linux-gnu.tgz --output core-agent-download.tgz
+
+# 3. Unzip the core-agent
+tar -xvzf core-agent-download.tgz
+
+# 4. Test that core agent is executable
+./core-agent
+</pre>
+    <p style='margin-top:60px;margin-bottom:10px'>
+    If everything has run successfully, you should see something similar to the following output:
+    </p>
+    <img src="images/core_agent_info.png" alt="core agent startup info/output"/>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <span class="step">3</span>
+      </td>
+      <td>
+        <p>Start the core agent:</p>
+<pre style='width:500px'>
+./core-agent start --socket ./core-agent.sock --daemonize true
+</pre>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <span class="step">4</span>
+      </td>
+      <td>
+        <p>Check to see that core agent socket is running:</p>
+<pre style='width:500px'>
+ ./core-agent probe --socket  ./core-agent.sock
+</pre>
+        <p style='margin-top:50px'>If you are using one of the supported languages (PHP, Python, Elixir, and Node.js), you will have to set the following configuration variables to point to the correct socket (as well as disabling the agent from re-downloading and launching the core agent again):<p>
+        <ul>
+            <li><code>SCOUT_CORE_AGENT_SOCKET_PATH=/var/www/scout_apm_core</code></li>
+            <li><code>SCOUT_CORE_AGENT_DOWNLOAD=false</code></li>
+            <li><code>SCOUT_CORE_AGENT_LAUNCH=false</code></li>
+        </ul>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Troubleshooting
+
+### Unable to launch Core Agent in tmp directory
```

Comment:
The users don't care about tmp directory in particular, other than it's the default we use.

This could be reframed as two distinct things:

1. "downloading the core agent to another directory", above the troubleshooting header
2. "checking if the core agent is executable" - here's the log message you'll see if we can't launch, try launching it manually as per <link>. Contact us if you continue having issues.

For 1, you can describe why: "You may want to do this if the default directory of /tmp/ doesn't work for your use case due to permissions, or if /tmp is mounted as noexec".

----

Also be careful referencing other sections, it'll lead to broken references, or referring to things that have been rewritten or moved.